docs/release-notes.md

```diff
@@ -26,9 +62,9 @@ The release notes include:
 ## **1.3.3** for Magento Commerce 2.3.6
 
 -  {:.fix}<!-- MC-32766 -->Fixed the Text content type to correctly save the Magento variable directives added to html attributes.
--  {:.fix}<!-- MC-34590 -->Fixed the Text content type to correctly handle and save non-Latin-1 characters.
--  {:.fix}<!-- MC-34677 -->Fixed Page Builder CSS styles that caused Safari 13.1 to incorrectly render the Luma theme's site menu for small view ports and mobile screens.
--  {:.fix}<!-- MC-34846 -->Fixed the HTML content type to correctly display embedded widgets like "Order by SKU" on the Storefront.
+-  {:.fix}<!-- MC-34590-->Fixed the Text content type to correctly handle and save non-Latin-1 characters.
```

Comment:
did you mean to do this? glancing at content above in this doc, you have some issue comments w/ leading spaces, some with both leading & trailing spaces, & some without either leading or trailing spaces. so i'm not sure what your shooting for here 😄